src/App.js

```diff
@@ -1,30 +1,41 @@
 import React, { Component } from 'react';
 import MovieContainer from './MovieContainer';
 import './App.css';
-import movieData from './MovieData';
 import Header from './Header';
 import SelectedMovie from './SelectedMovie';
 
 class App extends Component {
   constructor() {
     super()
     this.state = {
-      movies: movieData.movies
+      movies: [],
+      currentSection: 'mainPage',
+      selectedID: 0,
+      error: ''
     }
   }
 
-  componentDidMount() {
+  componentDidMount = () => {
     fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
     .then(res => res.json())
     .then(data => this.setState({movies: data.movies}))
+    .catch(err => {
+      this.setState({error: err})
+    });
+  }
+
+  changeSection = (section, id) => {
+    this.setState({currentSection: section, selectedID: id});
   }
 
   render() {
     return (
       <main className='App'>
         <Header />
```

Comment:
While I haven't looked at your Header component - I just love that you broke it out into its own component!